wiki/People/The_Team/Nomination_Assessment_Team/SEV_rating/en.md

```diff
@@ -0,0 +1,161 @@
+
+## Obviousness and Severity
+
+**Obviousness** relates to how easy it is to spot the issue.
+
+| Rating | Definition | Explanation |
+| :-- | :-- | :-- |
+| 0 | Not obvious | Applies if the issue is obscure or too granular to consistently catch on to. |
+| 1 | Can be found with experience | Requires knowledge / experience / attentiveness to catch. Can typically not be found by tools or regular users, e.g. timing / metadata issues. |
+| 2 | Can be found at a glance | Likely something that could be spotted by a regular user, or that wouldn’t have been missed when checking and using tools properly. |
+
+**Severity** relates to how much the issue impacts gameplay.
+
+| Rating | Definition | Explanation |
+| :-- | :-- | :-- |
+| 0 | Negligible | Doesn't or hardly affects gameplay. |
+| 1 | Worth noting | Impacts gameplay negatively but not significantly. |
+| 2 | Moderate design flaw | Harms gameplay to the point where it's typically also noticeable to a regular user, e.g. a big jump in a low difficulty. In practice, this is often a combination of multiple notable reasons, e.g. being both too difficult to read and an unwarranted difficulty spike. |
+
+Below are examples of SEV ratings and how they are roughly interpreted by evaluators:
+
+| SEV | Description |
+| :-- | :-- |
+| **0/0** | This reset is insignificant and ignored for the purpose of evaluations. |
+| **0/1** | Something went wrong here but it’s hard to blame the BNs, as it was difficult to spot. |
+| **1/0** | Could've been fixed if the BNs had been more attentive, but isn’t a significant issue. |
+| **1/1** | Something went wrong here that could have been fixed if the BNs were more attentive. |
+| **1/2** | Often means many things went wrong, but all require experience to spot easily. |
+| **2/0** | Some glaring issue in the map’s settings, like metadata, was somehow missed. |
+| **2/1** | Some glaring issue in the map's gameplay, like no hitsounding, was missed. |
+| **2/2** | An issue so severe that it is difficult not to spot, like concurrent hit objects, was missed. |
+
+## How SEV is used
+
+SEV ratings are used in the evaluations of the Beatmap Nominators, weighed against how many nominations they've done.
+
+Mistakes happen and a healthy amount will help with learning, but if they happen too frequently or the same mistakes repeat, then that's a problem. This is why evaluations don't focus on individual SEV ratings, but take the entire picture into account on a case-by-case basis.
+
+## Common reset reasons
+
+*Makes up 90% of resets.*
+
+Here is an exhaustive list of reasons nominations have been reset for with their respective SEV rating. These are based on stats of all SEV ratings set in the osu! game mode from February 2020 to April 2021, with percentages to show how common issues are.
```

Comment:
what does this first line mean? this?

```suggestion
Below is an exhaustive list of reasons nominations have been reset for with their respective SEV rating. These are based on statistics of all SEV ratings set in the osu! game mode from February 2020 to April 2021, with percentages to show how common issues are. The categories listed make up 90% of resets.
```

Reply:
it means that we've basically took 90% of all the resets in the timeframe mentioned below and analyzed them, I assume the other 10% are resets that are either too obscure or not worth mentioning, hence the "common reset reasons" section title